src/internal/deobfuscateAppleGPU.ts

```diff
@@ -82,7 +82,7 @@ export const deobfuscateAppleGPU = (
       gl.vertexAttribPointer(aPosition, 3, GL_FLOAT, false, 0, 0);
       gl.enableVertexAttribArray(aPosition);
 
-      gl.clearColor(1.0, 1.0, 1.0, 1.0);
+      gl.clearColor(1, 1, 1, 1);
```

Comment:
Seeing as all numbers are floats internally in JS it is fine to use 1 but the clearColor spec notes that the numbers must be GLclampf.